src/core/reference/include/openvino/reference/reduce_l1.hpp

```diff
@@ -31,6 +31,9 @@ void reduce_l1(InputIt in, OutputIt out, const Shape& in_shape, const AxisSet& r
 
     const auto out_shape = ov::util::reduce(in_shape, reduction_axes);
     std::fill(out, std::next(out, shape_size(out_shape)), T(0));
+    if (shape_size(in_shape) == 0) {
```

Comment:
How these changes impact on openvino library binary size?

Can it be reduced?

Reply:
Removed, where not needed.